src/components/Hero.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import React from 'react'
+import Leaderboard from './Leaderboard';
+import styles from '../styles/Hero.module.css'
+
+const Hero = () => {
+  return (
+    <>
+      <div className={styles.hero}>
+        <div className={styles.hero__leftcolumn}>
+          <div className={styles.hero__caption}>
+          <h1 className={styles.hero__title}>Isn’t it time you were rewarded for your epic skills?</h1>
```

Comment:
because this area is not currently a slide show, dynamic text is not required. 
the only component that is dynamic are the names and numbers in the leaderboard component.
does this change your instruction?

Reply:
added as const.